### ISSUE TITLE
make test can run integration & unit test suite in parallel

### ISSUE THREAD

**Issue by jhass** [ysbaddaden/shards#62]
And that blows up most of the time since one deletes directories the other creates or vice versa. One can workaround with `make -j1 test` but it would be nice to have a Makefile (or otherwise) setup that prevents running them at the same time in the first place.

**Comment by ysbaddaden** [comment]
`make test` always run sequentially for me (Linux and OS X), I never realized they could run in parallel :/

**Comment by jhass** [comment]
It's quite common to have `MAKEFLAGS="-j7"` or similar exported.
